tests/features/test_array_xd.py

```diff
@@ -359,6 +359,17 @@ def test_array_xd_with_np(data, feature, expected):
     assert ds[0]["col"] == expected
 
 
+@pytest.mark.parametrize(
+    "data, feature, expected",
+    [
+        (np.zeros((5000000, 768), dtype=np.float32), None, [0] * 768),
```

Comment:
This should be enough to pass the tests:

```suggestion
        (np.zeros((5000000, 768), dtype=np.uint8), None, [0] * 768),
```

(According to the GH docs, the Windows/Linux runner has 7GB of RAM)

Reply:
still failed while using uint8 :(

Reply:
Okay, let's try with this array: `np.zeros((46341, 46341), dtype=np.uint8)`  (it goes barely over the limit)